boards/xiao_m0/README.md

```diff
@@ -21,3 +21,18 @@ for examples.
   * Note that if you're using an older `cargo-hf2` that you'll need to specify
     the VID/PID when flashing: `cargo hf2 --vid 0x2886 --pid 0x002f --release
     --example blink --features="unproven"`
+
+If you are on Linux and hf2 fails to flash your board even if it is connected and in bootloader mode, you
+might need to add some `udev` rules if you have not done that yet.
+
+The example for adafruit boards is shown [here](https://crates.io/crates/hf2).
+
+You might want to have all the hf2 related rules in a single file, i.e. `/etc/udev/rules.d/99-hf2-boards.rules`, or have
+a different file for each vendor. The rules for Seeeduino boards look like this:
+
+```
+#seeeduino rules
+ATTRS{idVendor}=="2886", ENV{ID_MM_DEVICE_IGNORE}="1"
+SUBSYSTEM=="usb", ATTRS{idVendor}=="2886", MODE="0666"
+SUBSYSTEM=="tty", ATTRS{idVendor}=="2886", MODE="0666"
+```
```

Comment:
same `hf2` comment here.

Reply:
perhaps this comment since it seems more thorough should be the one copied around to the root README and the other README you modified.